src/server/database/getRethinkConfig.js

```diff
@@ -0,0 +1,25 @@
+import {readCert} from './readCert';
+import flag from 'node-env-flag';
+
+export const getRethinkConfig = () => {
+  const config = {
+    host: process.env.DATABASE_HOST || 'localhost',
+    port: process.env.DATABASE_PORT || 28015,
+    authKey: process.env.DATABASE_AUTH_KEY || '',
+    db: process.env.NODE_ENV === 'testing' ? 'actionTesting' : 'action',
+    min: process.env.NODE_ENV === 'production' ? 50 : 3,
+    buffer: process.env.NODE_ENV === 'production' ? 50 : 3
+  };
+
+  if (process.env.NODE_ENV && flag(process.env.DATABASE_SSL)) {
+    // we may need a cert for production deployment
+    // Compose.io requires this, for example.
+    // https://www.compose.io/articles/rethinkdb-and-ssl-think-secure/
```

Comment:
We've been 'running' on our rethinkdb container on dokku. I _love_ that we have the easy option to move to an externally hosted partner. Love compose.io. Love that you've made this easy by doing the fiddly work.